.github/workflows/pr-checks.yml

```diff
@@ -17,46 +17,51 @@ jobs:
         node-version: [16.x]
     steps:
     - uses: actions/checkout@v2
-    - uses: ./.github/actions/setup
-      id: setup
+    - uses: Khan/actions@shared-node-cache-v0
       with:
         node-version: ${{ matrix.node-version }}
 
+    - name: Get All Changed Files
+      uses: jaredly/get-changed-files@absolute
+      id: changed
+      with:
+        format: 'json'
+
     - id: js-files
       name: Find .js changed files
-      uses: ./.github/actions/filter-files
+      uses: Khan/actions@filter-files-v0
       with:
-        changed-files: ${{ steps.setup.outputs.changed_files }}
+        changed-files: ${{ steps.changed.outputs.added_modified }}
```

Comment:
Does `added_modified` include `renamed_and_modified` or is this going to miss testing files that were renamed and also modified?

No change requested for this PR since this question is related to the shared action rather than this specific change, but maybe there's work to be done in the shared action?

Reply:
hmmm that's a very good question. I can't tell from the github api docs (https://docs.github.com/en/rest/commits/commits#compare-two-commits) whether a file that was moved & then modified would show up as "renamed" or "modified" or "both" 🤔

Reply:
OK. I did some work in webapp to account for this in the meantime. You can see it in the static service pr workflow.

Reply:
There is a task in the backlog to add that to wonder blocks and wonder stuff.

Reply:
https://github.com/Khan/webapp/blob/bc196bdbdb3e9deaea68aac146fdb0211791a4b3/.github/workflows/static-service-pr-actions.yml#L50-L79